README.md

```diff
@@ -102,11 +102,13 @@ Each time you need to consume recent EDS changes elsewhere, you need to bump the
 - Prompt you for the new version number
 - Update `package.json` with the version number
 - Commit the change in a new commit
+- Get your PR approved in your branch and merge it to master
 
-2. To create and push a git tag, do one of these:
+2. To create and push a git tag:
 
-- Run `git tag v1.2.3 && git push origin v1.2.3` on your branch.
-- Create a release `v1.2.3` on GitHub, if it's already in trunk.
+- Switch to `master` branch and pull in your latest change
```

Comment:
I don't think this step is needed. As long as the last commit in your PR before it was merged, is the one where you ran `yarn version`, the tag should be on that commit. Pushing the tag doesn't require you be on any specific branch, does it?